autopub/publish_release.py

```diff
@@ -1,22 +1,17 @@
+import glob
 import os
 import sys
 
 sys.path.append(os.path.dirname(__file__))  # noqa
 
-from base import BUILD_SYSTEM, PYPI_URL, run_process
-
-poetry_pub = ["poetry", "publish", "-u", "$PYPI_USERNAME", "-p", "$PYPI_PASSWORD"]
-
-if PYPI_URL:
-    twine_pub = ["twine", "upload", "--repository-url", PYPI_URL, "dist/*"]
-else:
-    twine_pub = ["twine", "upload", "dist/*"]
-
-if BUILD_SYSTEM == "poetry":
-    pub_cmd = poetry_pub
-else:
-    pub_cmd = twine_pub
+from base import PYPI_URL, run_process
 
 
 def publish_release():
-    run_process(pub_cmd)
+    env = None
+    if PYPI_URL:
+        env = {"TWINE_REPOSITORY_URL": PYPI_URL}
+    dists = glob.glob("dist/*")
```

Comment:
Use glob here because `dist/*` never worked without a shell?

Reply:
I believe the Twine command always seemed to yield correct results, but either way I think your approach is superior. 👍